
modification of Huntingtin and Huntington’s disease pathology. Science 304, 100–104 25. Southwell, A. L., Smith-Dijak, A., Kay, C., Sepers, M., Villanueva, E. B., Parsons, M. P., Xie, Y., Anderson, L., Felczak, B., Waltl, S., Ko, S., Cheung, D., Cengio, L. D., Slama, R., Petoukhov, E., Raymond, L. A., and Hayden, M. R. (2016) An enhanced Q175 knock-in mouse model of Huntington disease with higher mutant huntingtin levels and accelerated disease phenotypes. Hum. Mol. Genet. 25, 3654–3675 26. Ochaba, J., Morozko, E. L., O’Rourke, J. G., and Thompson, L. M. (2018) Fractionation for resolution of soluble and insoluble huntingtin species. J. Vis. Exp. 132, e57082 27. Van Der Wal, L., Bezstarosti, K., Sap, K. A., Dekkers, D. H. W., Rijkers, E., Mientjes, E., Elgersma, Y., and Demmers, J. A. A. (2018) Improvement of ubiquitylation site detection by Orbitrap mass spectrometry. J. Proteomics 172, 49–56 28. Tyanova, S., Temu, T., and Cox, J. (2016) The MaxQuant computational platform for mass spectrometry – based shotgun proteomics. Nat. Protoc. 11, 2301–2319 29. The UniProt Consortium. (2017) UniProt: The universal protein knowledgebase. Nucleic Acids Res. 45, D158–D169 30. Tyanova, S., Temu, T., Sinitcyn, P., Carlson, A., Hein, M. Y., Geiger, T., Mann, M., and Cox, J. (2016) The Perseus computational platform for comprehensive analysis of (prote)omics data. Nat. Methods 13, 731–740 31. Doncheva, N. T., Morris, J. H., Gorodkin, J., and Jensen, L. J. (2019) Cytoscape StringApp: network analysis and visualization of proteomics data. J. Proteome Res. 18, 623–632 32. Shannon, P., Markiel, A., Owen Ozier 2, Baliga, N. S., Wang, J. T., Ramage, D., Amin, N., Schwikowski, B., and Ideker, T. (2003) Cytoscape: a software environment for integrated models of biomolecular interaction networks. Genome Res. 2498–2504 33. Maere, S., Heymans, K., and Kuiper, M. (2005) BiNGO: a Cytoscape plugin to assess overrepresentation of Gene Ontology categories in Biological Networks. Bioinformatics 21, 3448–3449 34. VIB-UGENT Center for Plant Systems Biology (Last accessed: 04/04/2019) Calculate and draw custom Venn diagrams. URL: http://bioinformatics. psb.ugent.be/webtools/Venn/ 

35. Bader, G. D., and Hogue, C. W. (2003) An automated method for finding molecular complexes in large protein interaction networks. BMC Bioinformatics 4, 2 36. O’Rourke, J. G., Gareau, J. R., Ochaba, J., Song, W., Rasko, T., Reverter, D., Lee, J., Monteys, A. M., Pallos, J., Mee, L., Vashishtha, M., Apostol, B. L., Nicholson, T. P., Illes, K., Zhu, Y. Z., Dasso, M., Bates, G. P., Difiglia, M., Davidson, B., Wanker, E. E., Marsh, J. L., Lima, C. D., Steffan, J. S., and Thompson, L. M. (2013) SUMO-2 and PIAS1 modulate insoluble mutant huntingtin protein accumulation. Cell Rep. 4, 362–375 37. Xie, Z., Adamowicz, W. O., Eldred, W. D., Jakowski, A. B., Kleiman, R. J., Morton, D. G., Stephenson, D. T., Strick, C. A., Williams, R. D., and Menniti, F. S. (2006) Cellular and subcellular localization of PDE10A, a striatum-enriched phosphodiesterase. Neuroscience 139, 597–607 38. Giampa` , C., Laurenti, D., Anzilotti, S., Bernardi, G., Menniti, F. S., and Fusco, F. R. (2010) Inhibition of the striatal specific phosphodiesterase PDE10A ameliorates striatal and cortical pathology in R6/2 mouse model of Huntington’s disease. PLoS ONE 5, 1–14 39. Hosp, F., Gutie´ rrez-A´ ngel, S., Schaefer, M. H., Cox, J., Meissner, F., Hipp, M. S., Hartl, F. U., Klein, R., Dudanova, I., and Mann, M. (2017) Spatiotemporal proteomic profiling of Huntington’s disease inclusions reveals widespread loss of protein function. Cell Rep. 21, 2291–2303 40. Skotte, N. H., Andersen, J. V., Santos, A., Nørremølle, A., Waagepetersen, H. S., Nielsen, M. L., Aldana, B. I., Willert, C. W., and Nielsen, M. L. (2018) Integrative characterization of the R6/2 mouse model of Huntington’s disease reveals dysfunctional astrocyte metabolism. Cell Rep. 23, 2211–2224 41. Rutherford, N. J., Lewis, J., Clippinger, A. K., Thomas, M. A., Adamson, J., Cruz, P. E., Cannon, A., Xu, G., Golde, T. E., Shaw, G., Borchelt, D. R., and Giasson, B. I. (2013) Unbiased screen reveals ubiquilin-1 and -highly associated with huntingtin inclusions. Brain Res. 1524, 62–73 42. Szklarczyk, D., Morris, J. H., Cook, H., Kuhn, M., Wyder, S., Simonovic, M., Santos, A., Doncheva, N. T., Roth, A., Bork, P., Jensen, L. J., and Von Mering, C. (2017) The STRING database in 2017: Quality-controlled protein-protein association networks, made broadly accessible. Nucleic Acids Res. 45, D362–D368 

43. Wickham, H. (2009) ggplot2: Elegant Graphics for Data Analysis. SpringerVerlag New York 44. RCore Team. (2018) R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria. https:// www.R-project.org/ 

45. Culver, B. P., Savas, J. N., Park, S. K., Choi, J. H., Zheng, S., Zeitlin, S. O., Yates, J. R., and Tanese, N. (2012) Proteomic analysis of wild-type and mutant huntingtin-associated proteins in mouse brains identifies unique interactions and involvement in protein synthesis. J. Biol. Chem. 287, 21599–21614 46. Russell, D. S., Barret, O., Jennings, D. L., Friedman, J. H., Tamagnan, G. D., Thomae, D., Alagille, D., Morley, T. J., Papin, C., Papapetropoulos, S., Waterhouse, R. N., Seibyl, J. P., and Marek, K. L. (2014) The phosphodiesterase 10 positron emission tomography tracer, [18F]MNI-659, as a novel biomarker for early huntington disease. JAMA Neurol. 71, 1520–1528 47. Langfelder, P., Cantle, J. P., Chatzopoulou, D., Wang, N., Gao, F., AlRamahi, I., Lu, X. H., Ramos, E. M., El-Zein, K., Zhao, Y., Deverasetty, S., Tebbe, A., Schaab, C., Lavery, D. J., Howland, D., Kwak, S., Botas, J., Aaronson, J. S., Rosinski, J., Coppola, G., Horvath, S., and Yang, X. W. (2016) Integrated genomics and proteomics define huntingtin CAG length-dependent networks in mice. Nat. Neurosci. 19, 623–633 48. Yau, R. G., Doerner, K., Castellanos, E. R., Matsumoto, M. L., Dixit, V. M., Rape, M., Haakonsen, D. L., Werner, A., Wang, N., Yang, X. W., and Martinez-Martin, N. (2017) Assembly and function of heterotypic ubiquitin chains in cell-cycle and protein quality control. Cell 171, 1–16 49. Hjerpe, R., Bett, J. S., Keuss, M. J., Knebel, A., Marchesi, F., Kurz, T., Hjerpe, R., Bett, J. S., Keuss, M. J., Solovyova, A., Mcwilliams, T. G., Glickman, M. H., Trost, M., Knebel, A., Marchesi, F., and Kurz, T. (2016) UBQLN2 mediates autophagy-independent protein aggregate clearance by the proteasome. Cell 166, 1–15 50. Franich, N. R., Basso, M., Andre, E. A., Ochaba, J., Kumar, A., Thein, S., Fote, G., Kachemov, M., Lau, A. L., Yeung, S. Y., Osmand, A., Zeitlin, S. O., Ratan, R. R., Thompson, L. M., and Steffan, J. S. (2018) Striatal mutant huntingtin protein levels decline with age in homozygous huntington’s disease knock-in mouse models. J. Huntingtons. Dis. 7, 137–150 51. Caviston, J. P., and Holzbaur, E. L. F. (2009) Huntingtin as an essential integrator of intracellular vesicular trafficking. Trends Cell Biol. 19, 147–155 52. Culver, B. P., Declercq, J., Dolgalev, I., Yu, M. S., Ma, B., Heguy, A., and Tanese, N. (2016) Huntington’s disease protein huntingtin associates with its own mRNA. J. Huntingtons. Dis. 5, 39–51 53. Savas, J. N., §1, M‡ Deinhardt, K., Culver, B. P., Restituito, S., Wu, L., Belasco, J. G., Chao, M. V., and Tanese, N. (2010) A role for Huntington disease protein in dendritic RNA granules. J. Biol. Chem. 285, 13142–13153 54. Hyrskyluoto, A., Lundh, S. H., Rappou, E., Petersen, A., Waltimo, T., Lindholm, D., Do, H. T., Reijonen, S., Bruelle, C., Kivinen, J., and Korhonen, L. (2014) Ubiquitin-specific protease-14 reduces cellular aggregates and protects against mutant huntingtin-induced cell degeneration: involvement of the proteasome and ER stress-activated kinase IRE1. Hum. Mol. Genet. 23, 5928–5939 55. Chhangani, D., Upadhyay, A., Amanullah, A., Joshi, V., and Mishra, A. (2014) Ubiquitin ligase ITCH recruitment suppresses the aggregation and cellular toxicity of cytoplasmic misfolded proteins. Sci. Rep. 4, 5077 56. Gu, M., Gash, M. T., Mann, V. M., Javoy-Agid, F., Cooper, J. M., and Schapira, A. H. V. (1996) Mitochondrial defect in Huntington’s disease caudate nucleus. Ann. Neurol. 39, 385–389 57. Browne, S. E., Bowling, A. C., MacGarvey, U., Baik, M. J., Berger, S. C., Muqit, M. M. K., Bird, E. D., and Beal, M. F. (1997) Oxidative damage and metabolic dysfunction in huntington’s disease: Selective vulnerability of the basal ganglia. Ann. Neurol. 41, 646–653 58. Bae, B. II, Xu, H., Igarashi, S., Fujimuro, M., Agrawal, N., Taya, Y., Hayward, S. D., Moran, T. H., Montell, C., Ross, C. A., Snyder, S. H., and Sawa, A. (2005) p53 mediates cellular dysfunction and behavioral abnormalities in Huntington’s disease. Neuron 47, 29–41 59. Akimov, V., Barrio-Hernandez, I., Hansen, S. V. F., Hallenborg, P., Pedersen, A. K., Bekker-Jensen, D. B., Puglia, M., Christensen, S. D. K., Vanselow, J. T., Nielsen, M. M., Kratchmarova, I., Kelstrup, C. D., Olsen, J. V., and Blagoev, B. (2018) UbiSite approach for comprehensive map