
In the insoluble fraction more increased fold changes at the protein level were observed as compared with the soluble fraction, which is most likely the result of the relatively large differences in components of these proteomes (aggregates versus no aggregates). Increased protein fold changes were also reflected in increased diGly site fold changes in several cases. 

## DISCUSSION
 

Overlap with Other Huntington’s Disease Studies—Several differentially expressed proteins in our datasets have been associated with HD before. For instance, Pde10a, a protein 

highly expressed in striatal medium spiny neurons (37), was found to be downregulated in affected brain regions of R6/2 mice by using a quantitative mass spectrometry approach (40). Further, Giampa et al. showed that inhibition of Pde10a alleviated neurological deficits and brain pathology in the HD mice model R6/2 (38). Pde10 expression profiling has also been suggested as a biomarker to follow early stages of HD progression (46). The actin-binding cytoskeletal protein Synaptopodin that is required for the formation of the spine apparatus was also found to be differentially expressed in R6/2 HD mice brain (39). Further, we found that Serpina3K 

FIG. 4. Huntington’s disease dependent changes in the global proteome and ubiquitinome of the insoluble fractions of Q175FDN mouse brain lysates. A, STRING network analysis was performed on differentially expressed proteins in the insoluble fraction. Orange is used for proteins identified exclusively in the mutant Htt samples (identified in 4 mutant samples and in 0 or 1 wild-type sample), whereas blue is used for proteins identified exclusively in wild-type samples (identified in 4 wild-type samples and in 0 or 1 mutant sample). A color gradient representing the protein fold change of orange (for upregulation in mutant) and blue (for upregulation in wild type) is used for significantly differentially expressed proteins (identified in at least 3 out of 4 mutant and wild-type samples, significant proteins in volcano plots Fig. 2B). B, Same as Fig. 4A, but for diGly sites. C, Gene Ontology (GO) analysis of differentially expressed proteins in the global proteome of the insoluble fraction. Top ten of enriched Biological Processes is shown, enrichment probability is represented by -log10 Benjamini Hochberg adjusted p value. D, Gene Ontology (GO) analysis of differentially expressed diGly sites in the insoluble fraction. Top ten of enriched Biological Processes is shown, enrichment probability is represented by -log10 Benjamini Hochberg adjusted p value. E, With the use of Cytoscape plugin MCODE highly interconnected clusters were found in the STRING network of the differentially expressed proteins in the insoluble fraction. Clusters with highest scores are shown here. See A for description of node color coding. F, same as E, but for diGly sites. 

log2(Global(Mutant/Wildtype)) 

Glul(95) 

Chgb(116) 

Eif3a(420) 

Htt(6) 

Ube2n(82) 

Vamp3(39) 

Dpp8(382) 

Sirt2(55) 

−4 

−2 

0 

2 

4 

−10 −8 −6 −4 −2 0 2 4 6 8 10 

log2(diGly(Mutant/Wildtype)) 

<image: DeviceRGB, width: 1, height: 20, bpc: 8>

1 2 3 4 5 6 log10(digly pval) 

Soluble 

Insoluble 

A 

B 

Hsp90aa1(112) 

Ubc(48) 

Calm3(31) 

Htt(6) 

Htt(9) 

Sqstm1(422) 

Sqstm1(437) 

−4 

−2 

0 

2 

4 

−8 −6 −4 −2 0 2 4 6 8 

log2(diGly(Mutant/Wildtype)) 

log2(Global(Mutant/Wildtype)) 

<image: DeviceRGB, width: 1, height: 20, bpc: 8>

1 2 3 4 log10(diGly pval) 

FIG. 5. Correlation between global proteome and ubiquitinome datasets in the soluble and insoluble fraction. Log2 fold change in global protein levels versus diGly sites are shown in these figures along with the density distributions of fold changes for both soluble (A) and insoluble (B) fractions. Each diGly site identified in at least 3 out of 4 replicates both in mutant and wild type is shown as a single data point with the corresponding diGly site fold change and the fold change of the protein that the diGly peptide is coming from. p values from the diGly t test are represented using a color gradient. The dashed y � x line and the dashed square are for ease of interpretation, latter representing the area with fold changes between �1 and 1. 