
was added to a final concentration of 1 mM. Proteins were digested for 1 h with Arg-C (Protea Biosciences, Morgantown, West Virginia; 1:200 enzyme/substrate ratio) at 37 °C and then overnight at room temperature (RT). Peptides were reduced using 5 mM DTT for 45 min at RT and subsequently alkylated using 5.5 mM chloroacetamide (CAA) for 30 min in the dark. CaCl2 was added to a final concentration of 1 mM. Samples were digested for 1 h with chymotrypsin (Pierce Chymotrypsin Protease, MS Grade, Thermo Scientific, Rockford, Illinois; 1:200 enzyme/substrate ratio) at 37 °C and then 3 h at RT. Trifluoroacetic acid (TFA) was added to 1% to stop the digestion. Digests were centrifuged at 4500 rpm for 15 min. Ten milligrams total protein of each sample was used for Lys-C/trypsin digestion. Samples were diluted with 50 mM Tris-HCl pH8 to a final concentration of 4 M urea. Proteins were reduced using 5 mM DTT for 45 min at RT and subsequently alkylated using 5.5 mM CAA for 30min in the dark. Proteins were digested for 2 h with Lys-C (Wako Pure Chemical Industries Ltd., Osaka, Japan; 1:100 enzyme/substrate ratio). Samples were diluted with 25 mM Tris-HCL pH8 to a final concentration of 1.6 M urea. CaCl2 was added to a final concentration of 1 mM. Samples were digested for 1h with trypsin (Pierce Trypsin Protease, MS-Grade, Thermo Scientific, Rockford, Illinois; 1:50 enzyme/substrate ratio) at 37 °C and then o/n at RT. TFA was added to 1% to stop the digestion. Digests were centrifuged at 4500 rpm for 15 min. Lys-C/trypsin Digestion of Insoluble Fractions—2.75 mg of each sample was used for Lys-C/trypsin digestion. Digestion was performed as described above. Peptide Fractionation of Soluble Fractions—Arg-C/chymotrypsin and Lys-C/trypsin digests of soluble fractions were combined. Peptide fractionation, diGly peptide enrichment and NanoLC-MS/MS analysis were performed as initially described (27), with only minor adjustments. Briefly, for peptide fractionation high pH reverse-phase (RP) chromatography was performed using polymeric PLRP-S (300Å, 50 �m, Agilent Technologies, part #PL1412–2K01). Samples were loaded onto the column and washed with 6 column volumes (CVs) of 0.1% TFA followed by 6 CVs of milliQ H2O. Peptides were then eluted in three fractions with 6 CVs of 10 mM ammonium formate solution (pH10) and 9%, 15 and 40% acetonitrile (AcN), respectively. 1/30 part of the fractions were dried to completeness by vacuum centrifugation and used for global proteome analysis. Remaining of the fractions were dried to completeness by lyophilization and used for diGly peptide enrichment. Peptide Clean-up of Insoluble Fractions—peptide clean-up was performed with Sep-Pak C18 columns (Waters, Dublin, Ireland) according to PTMscan protocol (PTMscan, Cell Signaling Technologies, Leiden, The Netherlands). Immunoprecipitation of diGly Peptides—Ubiquitin remnant motif (K-�-GG) antibodies coupled to beads (PTMscan, Cell Signaling Technologies) were used according to manufacturer’s advice. For the soluble fraction, each standard batch of beads was divided over the three fractions of each sample. For the insoluble fraction, 20 �l beads were used per diGly IP. NanoLC-MS—Mass spectra were acquired on an Orbitrap Tribrid Lumos mass spectrometer (Thermo Fisher Scientific GmbH, Bremen, Germany) coupled to an EASY-nLC 1200 system (Thermo). Peptides were separated on an in-house packed 75 �m inner diameter column containing 50 cm Waters CSH130 resin (2.5 �m, 130Å, Waters, Etten-Leur, The Netherlands) with a gradient consisting of 2–30% (AcN, 0.1%FA) over 120 min at 300 nL/min. The column was kept at 50 °C in a NanoLC oven - MPI design (Sonation GmbH, Biberach, Germany). For all experiments, the instrument was operated in the data-dependent acquisition (DDA) mode using the top speed method with a cycle time of 3 s. MS1 spectra were collected at a resolution of 120,000 with an automated gain control (AGC) target of 4E5 and a max injection time of 50 ms. Precursors were filtered according to 

charge state (2–6z), and monoisotopic peak assignment. Previously interrogated precursors were dynamically excluded for 60 s. Peptide precursors were isolated with a quadrupole mass filter set to a width of 1.6 Th. Ion trap MS2 spectra were collected at an AGC of 7E3, a max injection time of 50 ms and HCD collision energy of 30%. Duration of all individual mass spec runs was 120 min. Data Processing and Statistical Analysis—RAW files were analyzed using MaxQuant software suite (v1.6.5.0) (28). The data were searched against reviewed (Swiss-Prot) Mus musculus proteome from Uniprot (September 2018, contains 16981 entries) (29), along with revert decoys and standard contaminants database from MaxQuant. Methionine oxidation and N-terminal acetylation were set as variable modifications, whereas carbamidomethylation of cysteine was set as a fixed modification. In addition to these, diGly modification of Lysine was set as a variable modification for searching diGly data. Maximum number of modifications per peptide were set to 5. Default search settings for Orbitrap were used, such as a precursor tolerance of 20 ppm and 4.5 ppm for the first and main search respectively and a tolerance of 20 ppm for the fragment ions. Trypsin was selected as the search enzyme for the insoluble data; whereas Arg-C/chymotrypsin/trypsin was set as the cleavage enzyme for the soluble. Maximum number of miscleavages were set to 3. The minimal peptide length was set to 7, “second peptides” and “match between runs” were checked. The label-free quantification algorithm (LFQ) in MaxQuant was used for quantification with LFQ minimum ratio count set to 1; unique and razor peptides were selected to be used for quantification. The false discovery rate for peptide, protein, and site identifications were set to 1%. The minimum score for diGly peptides was kept at 40 (default). Statistical analyses were performed on MaxQuant outputs using the Perseus package (v1.6.2.3) (30). For global proteome data the proteinGroups files were used for statistical analyses. Proteins that were identified by less than 2 unique peptides were excluded. LFQ intensities were log2 transformed. Proteins that were identified in at least 3 (out of 4) replicates in both wtHtt and mHtt samples were selected for comparison of protein abundance. Missing values of selected proteins were then imputed from a normal distribution with width 0.3, down shift 1.8 from the general distribution. For ubiquitinome data the GlyGly (K) Sites tables were used for statistical analyses. diGly site modifications localized on C-term with a probability more than 0.75 and on any other position with a probability less than 0.75 were discarded. diGly sites that were identified in at least 3 (out of 4) experiments in both wtHtt and mHtt samples were selected for diGly site quantification. Normalization of diGly log2 transformed intensities was done manually based on average median intensities in the global evidence file. Calculated scaling factors were then applied to intensities in the GlyGly (K) sites table. In addition to these comparisons, proteins and diGly sites that were identified in all of the mHtt replicates but in none or only one of the wtHtt, and vice versa, were extracted from the tables before imputation and filtering for further exploration. Pathway Analysis—Protein-protein association network for up and down regulated proteins was visualized using the StringApp (v1.4.1) (31) in Cytoscape (v3.7.0) (32). Cytoscape plugin BiNGO (v3.0.3) (33) was used for identification of enriched GO terms from lists of gene names of up and down regulated proteins or diGly sites (target list) versus whole annotation as a reference set. Hypergeometric statistical test was used with Benjamini & Hochberg false discovery rate correction with a significance level of 0.05. Web-based Venn diagram tool (34) from VIB/UGhent was used to plot the overlap of up- and downregulated proteins in both the soluble and insoluble fractions. Cytoscape plugin Molecular Complex Detection (MCODE) (v1.5.1) (35) was used to identify highly interconnected clusters. The following settings were used for cluster identification: Network scoring: Degree 