
# Global Proteome and Ubiquitinome Changes in the Soluble and Insoluble Fractions of Q175 Huntington Mice Brains*
□ S 

Karen A. Sap‡, 

<image: DeviceRGB, width: 33, height: 33, bpc: 8>

Arzu Tugce Guler‡, Karel Bezstarosti§, Aleksandra E. Bury‡, Katrin Juenemann¶**, 

<image: DeviceRGB, width: 33, height: 33, bpc: 8>

Jeroen A. A. Demmers§, and Eric A. Reits‡� 

Huntington’s disease is caused by a polyglutamine repeat expansion in the huntingtin protein which affects the function and folding of the protein, and results in intracellular protein aggregates. Here, we examined whether this mutation leads to altered ubiquitination of huntingtin and other proteins in both soluble and insoluble fractions of brain lysates of the Q175 knock-in Huntington’s disease mouse model and the Q20 wild-type mouse model. Ubiquitination sites are detected by identification of Gly-Gly (diGly) remnant motifs that remain on modified lysine residues after digestion. We identified K6, K9, K132, K804, and K837 as endogenous ubiquitination sites of soluble huntingtin, with wild-type huntingtin being mainly ubiquitinated at K132, K804, and K837. Mutant huntingtin protein levels were strongly reduced in the soluble fraction whereas K6 and K9 were mainly ubiquitinated. In the insoluble fraction increased levels of huntingtin K6 and K9 diGly sites were observed for mutant huntingtin as compared with wild type. Besides huntingtin, proteins with various roles, including membrane organization, transport, mRNA processing, gene transcription, translation, catabolic processes and oxidative phosphorylation, were differently expressed or ubiquitinated in wild-type and mutant huntingtin brain tissues. Correlating protein and diGly site fold changes in the soluble fraction revealed that diGly site abundances of most of the proteins were not related to protein fold changes, indicating that these proteins were differentially ubiquitinated in the Q175 mice. In contrast, both the fold change of the protein level and diGly site level were increased for several proteins in the insoluble fraction, including ubiquitin, ubiquilin-2, sequestosome-1/p62 and myo5a. Our data sheds light on putative novel proteins involved in different cellular processes as well as their ubiquitination status in Huntington’s disease, which forms the basis for further mechanistic studies to understand the role of differential ubiquitination of huntingtin and ubiquitin-regulated processes in Huntington’s disease. Molecular & Cellular Proteomics 18: 1705–1720, 2019. DOI: 10.1074/mcp.RA119.001486. 

Huntington’s disease (HD)1 is an autosomal dominant inherited neurodegenerative disorder characterized by excessive motor movements, cognitive and psychiatric deficits (1). The disease is hallmarked by intracellular aggregates and neuronal loss in various brain regions, with the striatum being mostly affected by neurodegeneration (2, 3). Patients usually live for 15–20 years after the onset of the disease (4). HD is caused by an expansion of the CAG repeat present in the exon1 domain of the huntingtin (Htt) gene. Healthy individuals exhibit between 6 to 35 CAG repeats in their Htt N termini, whereas individuals affected by HD display �39 repeats. CAG repeats are translated into polyglutamine (polyQ) repeats and mutant Htt (mHtt) proteins are thus characterized by an N-terminal polyQ-repeat expansion, as compared with the wild-type Htt (wtHtt) protein. Experimental data and the dominant inheritance pattern of HD indicate that mHtt is toxic and triggers neurodegeneration (5). Htt can be cleaved by proteases, including caspases, into shorter fragments (6, 7). Especially N-terminal fragments of the mHtt protein are aggregation-prone, resulting in proteotoxicity and protein aggregation (8). Various studies have suggested that post-translational modifications (PTMs) of polyQ-expanded Htt modulate aggregation and toxicity (9, 10). Htt can be phosphorylated, SUMOylated, myristoylated (11) and acetylated on various residues, with phosphorylation being mostly studied. Interestingly, it has been shown that intracellular localization, cleavage, and degradation of mHtt are affected by phosphorylation of threonine 3 (12), serine 13, and serine 16, which are positioned in the N-terminal domain of Htt consisting of the first 17 amino acids (N17). Phosphorylation of these residues has also been shown to influence other post-translational modifications (13, 14). Identification of new PTMs may lead to effective therapeutic strategies, as involved enzymes may be modulated to improve the clearance of mHtt before cell toxicity. 

From the ‡Department of Medical Biology, Amsterdam UMC, location AMC, Meibergdreef 15, 1105 AZ Amsterdam, The Netherlands; §Department of Biochemistry, Erasmus University Medical Center, Wytemaweg 80, 3015 CN Rotterdam, The Netherlands; ¶Leibniz-Forschungsinstitut fu¨ r Molekulare Pharmakologie im Forschungsverbund Berlin, Robert-Roessle-St. 10 13089 Berlin, Germany Received April 5, 2019, and in revised form, May 21, 2019 Published, MCP Papers in Press, May 28, 2019, DOI 10.1074/mcp.RA119.001486 