
follow-up studies could include a cut-off filter to separate proteins based on size. Understanding the role of differential ubiquitination of huntingtin as well as ubiquitin-regulated processes in Huntington’s disease might lead to the identification of candidates for drug development with the aim to enhance the efficiency of degradation of toxic mHtt species via the Ubiquitin-Proteasome System. 

## Acknowledgment
—We thank Alicia Sanz Sanz for performing experiments for supplemental Fig. S4. 

## Data Availability
 

The mass spectrometry proteomics data have been deposited to the ProteomeXchange Consortium via the PRIDE (65) partner repository with the dataset identifier PXD010161. http://www.ebi.ac.uk/pride/archive/projects/PXD010161. 

* This work was carried out on the Dutch national e-infrastructure with the support of SURF Cooperative. This work was funded by CHDI and Campagneteam Huntington. ** Present address: Bayer AG Research & Development, Pharmaceuticals, Berlin, Germany. In the discussion section we refer to a manuscript in preparation by K. Juenemann. □ S This article contains supplemental Figures and Tables. � To whom correspondence should be addressed. Tel.: �31 (0)20 566 6259; E-mail: e.a.reits@amc.uva.nl. Author contributions: K.A.S. designed research; K.A.S. and A.E.B. performed research; K.A.S. and A.T.G. analyzed data; K.A.S., A.T.G., K.J., J.D., and E.R. wrote the paper; K.B. mass spectrometry operator. 

## REFERENCES
 

1. MacDonald, M. E., Ambrose, C. M., Duyao, M. P., Myers, R. H., Lin, C., Srinidhi, L., Barnes, G., Taylor, S. A., James, M., Groot, N., MacFarlane, H., Jenkins, B., Anderson, M. A., Wexler, N. S., Gusella, J. F., Bates, G. P., Baxendale, S., Hummerich, H., Kirby, S., North, M., Youngman, S., Mott, R., Zehetner, G., Sedlacek, Z., Poustka, A., Frischauf, A. M., Lehrach, H., Buckler, A. J., Church, D., Doucette-Stamm, L., O’Donovan, M. C., Riba-Ramirez, L., Shah, M., Stanton, V. P., Strobel, S. A., Draths, K. M., Wales, J. L., Dervan, P., Housman, D. E., Altherr, M., Shiang, R., Thompson, L., Fielder, T., Wasmuth, J. J., Tagle, D., Valdes, J., Elmer, L., Allard, M., Castilla, L., Swaroop, M., Blanchard, K., Collins, F. S., Snell, R., Holloway, T., Gillespie, K., Datson, N., Shaw, D., and Harper, P. S. (1993) A novel gene containing a trinucleotide repeat that is expanded and unstable on Huntington’s disease chromosomes. Cell 72, 971–983 2. Graveland, G. A., Williams, R. S., and DiFiglia, M. (1985) Evidence for degenerative and regenerative changes in neostriatal spiny neurons. Science 227, 770–773 3. Vonsattel, J. P. G., and DiFiglia, M. (1998) Huntington disease. J. Neuropathol. Exp. Neurol. 57, 369–384 4. Ross, C. A., and Tabrizi, S. J. (2011) Huntington’s disease: From molecular pathogenesis to clinical treatment. Lancet Neurol. 10, 83–98 5. Imarisio, S., Carmichael, J., Korolchuk, V., Chen, C.-W., Saiki, S., Rose, C., Krishna, G., Davies, J. E., Ttofi, E., Underwood, B. R., and Rubinsztein, D. C. (2008) Huntington’s disease: from pathology and genetics to potential therapies. Biochem. J. 412, 191–209 6. Kim, Y. J., Yi, Y., Sapp, E., Wang, Y., Cuiffo, B., Kegel, K. B., Qin, Z. H., Aronin, N., and DiFiglia, M. (2001) Caspase 3-cleaved N-terminal fragments of wild-type and mutant huntingtin are present in normal and Huntington’s disease brains, associate with membranes, and undergo calpain-dependent proteolysis. Proc. Natl. Acad. Sci. U.S.A. 98, 12784–12789 7. Graham, R. K., Deng, Y., Slow, E. J., Haigh, B., Bissada, N., Lu, G., Pearson, J., Shehadeh, J., Bertram, L., Murphy, Z., Warby, S. C., Doty, 

C. N., Roy, S., Wellington, C. L., Leavitt, B. R., Raymond, L. A., Nicholson, D. W., and Hayden, M. R. (2006) Cleavage at the caspase-6 site is required for neuronal dysfunction and degeneration due to mutant huntingtin. Cell 125, 1179–1191 8. Ratovitski, T., Gucek, M., Jiang, H., Chighladze, E., Waldron, E., D’Ambola, J., Zhipeng, H., Yideng, L., Poirier, M. A., Hirschhorn, R. R., Graham, R., Hayden, M. R., Cole, R. N., and Ross, C. A. (2009) Mutant huntingtin N-terminal fragments of specific size mediate aggregation and toxicity in neuronal cells. J. Biol. Chem. 284, 10855–10867 9. Pennuto, M., Palazzolo, I., and Poletti, A. (2009) Post-translational modifications of expanded polyglutamine proteins: Impact on neurotoxicity. Hum. Mol. Genet. 18, R40–R49 10. Wang, Y., Lin, F., and Qin, Z.-H. (2010) The role of post-translational modifications of huntingtin in the pathogenesis of Huntington’s disease. Neurosci. Bull. 26, 153–162 11. Martin, D. D. O., Heit, R. J., Yap, M. C., Davidson, M. W., Hayden, M. R., and Berthiaume, L. G. (2014) Identification of a post-translationally myristoylated autophagy-inducing domain released by caspase cleavage of huntingtin. Hum. Mol. Genet. 23, 3166–3179 12. Aiken, C. T., Steffan, J. S., Guerrero, C. M., Khashwji, H., Lukacsovich, T., Simmons, D., Purcell, J. M., Menhaji, K., Zhu, Y. Z., Green, K., LaFerla, F., Huang, L., Thompson, L. M., and Marsh, J. L. (2009) Phosphorylation of threonine 3: Implications for huntingtin aggregation and neurotoxicity. J. Biol. Chem. 284, 29427–29436 13. Ehrnhoefer, D. E., Sutton, L., and Hayden, M. R. (2011) Small changes, big impact: Posttranslational modifications and function of huntingtin in Huntington disease. Neuroscientist. 17, 475–492 14. Thompson, L. M., Aiken, C. T., Kaltenbach, L. S., Agrawal, N., Illes, K., Khoshnan, A., Martinez-Vincente, M., Arrasate, M., O’Rourke, J. G., Khashwji, H., Lukacsovich, T., Zhu, Y. Z., Lau, A. L., Massey, A., Hayden, M. R., Zeitlin, S. O., Finkbeiner, S., Green, K. N., LaFerla, F. M., Bates, G., Huang, L., Patterson, P. H., Lo, D. C., Cuervo, A. M., Marsh, J. L., and Steffan, J. S. (2009) IKK phosphorylates Huntingtin and targets it for degradation by the proteasome and lysosome. J. Cell Biol. 187, 1083–1099 15. Wang, B., Zeng, L., Merillat, S. A., Fischer, S., Ochaba, J., Thompson, L. M., Barmada, S. J., Scaglione, K. M., and Paulson, H. L. (2018) The ubiquitin conjugating enzyme Ube2W regulates solubility of the Huntington’s disease protein, huntingtin. Neurobiol. Dis. 109, 127–136 16. de Pril, R., Fischer, D. F., Roos, R. A. C., and van Leeuwen, F. W. (2007) Ubiquitin-conjugating enzyme E2–25K increases aggregate formation and cell death in polyglutamine diseases. Mol. Cell Neurosci. 34, 10–19 17. Mishra, A., Dikshit, P., Purkayastha, S., Sharma, J., Nukina, N., and Jana, N. R. (2008) E6-AP promotes misfolded polyglutamine proteins for proteasomal degradation and suppresses polyglutamine protein aggregation and toxicity. J. Biol. Chem. 283, 7648–7656 18. Miller, V. M., Nelson, R. F., Gouvion, C. M., Williams, A., Rodriguez-Lebron, E., Harper, S. Q., Davidson, B. L., Rebagliati, M. R., and Paulson, H. L. (2005) CHIP suppresses polyglutamine aggregation and toxicity in vitro and in vivo. J. Neurosci. 25, 9152–9161 19. Luo, H., Cao, L., Liang, X., Du, A., Peng, T., and Li, H. (2018) Herp promotes degradation of mutant huntingtin: involvement of the proteasome and molecular chaperones. Mol. Neurobiol. 55, 7652–7668 20. Yang, H., Zhong, X., Ballar, P., Luo, S., Shen, Y., Rubinsztein, D. C., Monteiro, M. J., and Fang, S. (2007) Ubiquitin ligase Hrd1 enhances the degradation and suppresses the toxicity of polyglutamine-expanded huntingtin. Exp. Cell Res. 313, 538–550 21. Bhat, K. P., Yan, S., Wang, C., Li, S., and Li, X. (2014) Differential ubiquitination and degradation of huntingtin fragments modulated by ubiquitinprotein ligase E3A. Proc. Natl. Acad. Sci. U.S.A. 111, 5706–5711 22. Koyuncu, S., Saez, I., Lee, H. J., Gutierrez-Garcia, R., Pokrzywa, W., Fatima, A., Hoppe, T., and Vilchez, D. (2018) The ubiquitin ligase UBR5 suppresses proteostasis collapse in pluripotent stem cells from Huntington’s disease patients. Nat. Commun. 9, 1–9 23. Juenemann, K., Wiemhoefer, A., and Reits, E. a. (2015) Detection of ubiquitinated huntingtin species in intracellular aggregates. Front. Mol. Neurosci. 8, 1–8 24. Steffan, J. S., Agrawal, N., Pallos, J., Rockabrand, E., Trotman, L. C., Slepko, N., Illes, K., Lukacsovich, T., Zhu, Y.-Z., Cattaneo, E., Pandolfi, P. P., Thompson, L. M., and Marsh, J. L. (2004) SUMO 