
cutoff of 2. Cluster finding: haircut. Node Score cutoff: 0.2. K-core: 2. Max. Depth: 100. SDS-PAGE and Western Blot Analysis—Antibodies used for SDSPAGE WB analysis: �-Htt (ab109115 Abcam); �-Htt D7F7 (#5656 CST), �-Actin (SC-1616 Santa Cruz), �-Vimentin (AB1620 Sigma). SDS-PAGE and Western blotting were performed according to standard procedures, except for Htt (350 kDa). For Htt a 6% Tris/Glycine gel was used and proteins were run out of the gel until �80 kDa. Wet Blotting was done o/n at 45V at 4 °C. Proteins were visualized using LiCOR secondary antibodies and the Odyssey system. 

## RESULTS
 

Proteomics Workflow—The N-terminal sequence of Htt is subjected to a variety of post-translational modifications (PTMs), such as ubiquitination (23, 24), SUMOylation (24, 36) and phosphorylation (12, 14). We aimed to identify endogenous fl Htt ubiquitination sites in mice brain lysates. We were especially interested in the putative ubiquitination sites located at the N terminus; K6, K9, and K15, as the N17 domain (consisting of the first 17 amino acids preceding the polyQ repeat) plays a crucial role in mHtt localization, aggregation and clearance. In addition to Lys-C and trypsin digestion, Arg-C and chymotrypsin digestion was used to potentially increase the number of unique N-terminal Htt diGly peptides (Fig. 1A, supplemental Fig. S1). Four biological replicates of both wild-type mice brain lysates and Q175FDN mutant Htt mice brain lysates were processed for mass spectrometric analysis. Fig. 1B shows a simplified overview of the experimental workflow for one sample. Frozen brain tissues were grinded in liquid nitrogen. To prevent resolving mHtt fibrils, aggregates and inclusion bodies, samples were lysed in a 1%Triton X-100-based mild lysis buffer (26). After sonication, a Triton X-100 soluble and insoluble fraction were obtained through centrifugation. In our workflow the soluble Htt fraction contains wtHtt- or mHtt monomers but could also contain some oligomeric states of mHtt (26). Insoluble pellet fractions were treated with formic acid to solubilize Htt aggregates. Proteins of both the soluble and insoluble fractions were denatured in an 8 M urea-based buffer. Twenty milligrams and 3 mg of total protein were digested for the soluble and insoluble fractions, respectively. Recently, we have optimized the protocol for diGly peptide detection for samples with relatively high protein amounts (27). Briefly, in this workflow digests are subjected to offline high pH reverse phase (RP) fractionation. Three fractions were collected of each sample, and of each fraction 1/30 (v/v) part was subjected to global proteome analysis, whereas the remaining part was subjected to diGly peptide enrichment. All fractions were measured individually by nanoLC-MS/MS (Fig. 1B). Quantitative Analysis of Global Proteome and Ubiquitinome Data Sets—Raw files obtained for the global proteome and ubiquitinome of the soluble and insoluble fractions were analyzed using the MaxQuant and Perseus platforms. We identified 5229 protein groups in the global proteome data of the soluble fraction (supplemental Table S1), and 2590 protein 

groups in the global proteome data of the insoluble fraction (supplemental Table S2) (excluding contaminants and decoys). The proteins were ordered by the number of identified peptides in descending order in the protein IDs column of the proteinGroups table. The first protein, thus the protein with the highest number of identified peptides, was used from each protein group. For the diGly data we identified 5508 diGly peptides in the soluble fraction and 1146 diGly peptides in the insoluble fraction (excluding contaminants and decoys). DiGly sites were used for further analyses (supplemental Table S3 and supplemental Table S4). Hierarchical clustering (Euclidean distance) analysis was performed for all datasets and as expected we observed more variation in the ubiquitinome datasets as compared with the proteome datasets. The ubiquitinome of the insoluble fraction showed the most variation (Fig. 2A). Two-sample Student’s t-tests were performed for all proteins and diGly sites that were identified in at least 3 out of 4 samples in both wild-type and mutant conditions (Fig. 2B). The relative abundances of most of the proteins and diGly sites were unaffected. Proteins and diGly sites with significant higher relative abundances in the mutant samples are shown in orange, whereas proteins and diGly sites present with significant higher relative abundances in the wild-type samples are shown in blue. For informative purposes several differentially expressed proteins and diGly sites are depicted with their gene names in the plots. Differential expression of several proteins had been associated with HD before, such as downregulation of Pde10a (37–40), synaptopodin (39), Itpka (40), and Serpina3k (40), and upregulation of Mri1 (40) in HD mice models. Also, several proteins that we found to be enriched in the insoluble fraction of the mutant Htt samples have been identified in Htt aggregates in other studies, for instance Ina, Vim, Tra2a (39), and ubqln2 (41). Further, we observed a remarkable downregulation of the Htt protein in the soluble fraction of mHtt brain lysates (Fig. 2B upper plot). Changes in protein abundances observed by mass spectrometry were confirmed by SDS-PAGE in combination with Western blotting. For instance, we observed relatively lower levels of full-length mHtt in the soluble fraction and relatively higher levels of Vimentin in the insoluble fraction by SDS-PAGE WB (Fig. 2C). Additionally, we observed the presence of both full-length mHtt as well as oligomeric species and/or fragments of the full-length protein in the insoluble fraction (Fig. 2C). Although the selection of proteins identified in 3 out of 4 samples in both groups was a good basis for comparison between two groups, several putatively interesting candidates could have been omitted by using this method. In our study we focused on a late stage of HD, in which extreme differences might appear both at the protein level as well as at the level of protein ubiquitination. We therefore also investigated these extreme differences by including proteins or diGly sites identified in 4 out of 4 replicates in one group while identified exclusively in one or none of the replicates in the other group. 